src/components/Molecules/SearchResult/SearchResult.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import Text from '../../Atoms/Text/Text';
+import Picture from '../../Atoms/Picture/Picture';
+
+/**
+ * Search Result
+ */
+const Wrapper = styled.article`
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+`;
+
+const Link = styled.a`
+  display: flex;
```

Comment:
Can we use  `Link` component instead here ?

Reply:
done